scalableminds/webknossos
# Persist the selected overwrite-mode for hybrids and volume tracings

### URL of deployed dev instance (used for testing):
- https://rememberoverwritemode.webknossos.xyz

### Steps to test:
- open a volume or hybrid tracing
- select the brush tool
- verify that the different overwrite behaviors work (can be switched in the toolbar)
- reload the page and check that the last overwrite behavior is active after the reload

### Issues:
- fixes #4894

------
